conbench/tests/api/test_compare.py

```diff
@@ -9,7 +9,7 @@
 from ...tests.api import _asserts, _fixtures
 from ...tests.helpers import _uuid
 
-CASE = "snappy, cpu_count=2, parquet, arrow, nyctaxi_sample"
+CASE = "compression=snappy, cpu_count=2, dataset=nyctaxi_sample, file_type=parquet, input_type=arrow"
```

Comment:
Here we see how a case permutation is stringified now. See #400.

Reply:
Note to self: fix `benchalerts` tests to deal with this change to the `/api/compare` endpoint (the "benchmark" key contains this stringification), and of course longer term #720 is very important :)